zero_bin/leader/Cargo.toml

```
@@ -38,3 +39,4 @@ test_only = ["ops/test_only", "prover/test_only"]
 [build-dependencies]
 cargo_metadata = "0.18.1"
 anyhow = { workspace = true }
+vergen = { version = "9.0.0", features = ["build", "rustc"] }
```

This is specified multiple time in individual config files. Instead it should be defined at the workspace level and imported as
```suggestion
vergen = { workspace = true }
```